src/components/Navbar.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from "react";
+import "../styles/Nav.scss";
+import SearchRoundedIcon from "@material-ui/icons/SearchRounded";
+import HomeOutlinedIcon from "@material-ui/icons/HomeOutlined";
+import PeopleAltOutlinedIcon from "@material-ui/icons/PeopleAltOutlined";
+import NotificationsNoneOutlinedIcon from "@material-ui/icons/NotificationsNoneOutlined";
+import ExitToAppOutlinedIcon from "@material-ui/icons/ExitToAppOutlined";
+import PropTypes from "prop-types";
+import logo from "../images/logo.svg";
+
+// Example usage: <Navbar selected="Friends" />
+class NavBar extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: "Username",
+      notification: 1,
+    };
+  }
+  // will implement search later. Disabled eslint for this method for now
+
+  // eslint-disable-next-line class-methods-use-this
+  handleSubmit(event) {
+    if (event.key === "Enter") {
+      // eslint-disable-next-line no-alert
+      alert(event.target.value);
+    }
+  }
+
+  render() {
+    const { username, notification } = this.state;
+    const { selected } = this.props;
+    return (
+      <div className="nav">
+        <div className="navRow">
+          <div className="left-menu">
+            <div className="logo-container">
+              <img src={logo} width="70%" alt="app logo" />
+            </div>
+            <div className="search-input-container">
+              <SearchRoundedIcon style={{ color: "#0275B1" }} />
```

Comment:
not important but we should try to move this to the scss file instead of putting styles here. And then maybe we can make a sass variable `$accent_colour` or something since this colour is being used in multiple places